File: week3-exercises03-munit-tests/src/test/scala/com/datastax/workshop/Ex07_Query4a_ListJourneys.scala
package com.datastax.workshop

class Ex07_Query4a_ListJourneys extends ExerciseBase("Exercise4") {

  test("list journeys") {

    journeyRepo.findJourneys(SPACECRAFT).foreach { journey =>
      LOGGER.info("- Journey: {} Summary: {}", journey.id, journey.summary)
    }

    LOGGER.info("SUCCESS")
  }
}

File: week3-exercises01-java/src/test/java/com/datastax/workshop/Ex07_Query4a_ListJourneys.java
package com.datastax.workshop;

import java.util.List;

import org.junit.jupiter.api.Test;
import org.junit.platform.runner.JUnitPlatform;
import org.junit.runner.RunWith;

import com.datastax.oss.driver.api.core.cql.ResultSet;
import com.datastax.oss.driver.api.core.cql.Row;
import com.datastax.oss.driver.api.core.cql.SimpleStatement;

@RunWith(JUnitPlatform.class)
public class Ex07_Query4a_ListJourneys extends ExerciseBase {

    public Ex07_Query4a_ListJourneys() {
        super("Exercise4");
    }

    @Test
    public void listJourneys() {

        List<Journey> journeys = journeyRepo.findJourneys(SPACECRAFT);

        for (Journey journey : journeys) {
            LOGGER.info("- Journey: {} Summary: {}", journey.getId(), journey.getSummary());
        }

        LOGGER.info("SUCCESS");
    }
}
